doc/nrf/ug_nrf5340.rst

```diff
@@ -106,165 +85,246 @@ Zephyr includes the `OpenAMP`_ library, which provides a complete solution for e
 
-.. _ug_nrf5340_building:
+Therefore, the application core must always be programmed, even if the firmware is supposed to run only on the network core.
+You can use the :ref:`nrf5340_empty_app_core` sample for this purpose.
+This sample does not automatically include the network core application as child image, so you must either configure it appropriately or program the images for the network core and application core separately.
+See :ref:`ug_nrf5340_multi_image` for more information.
 
-Building and programming a sample
-*********************************
 
-Depending on the sample, you must program only the application core (for example, when using NFC samples) or both the network and the application core.
+No radio communication
+======================
+.. list-table::
+   :header-rows: 1
+
+   * - Network core
+     - Application core
+   * - ---
+     - | :ref:`NFC samples <nfc_samples>`
+       | :ref:`Crypto samples <crypto_samples>`
+       | :ref:`tfm_hello_world`
+       | :ref:`lpuart_sample`
+
 
 .. note::
-   On nRF5340, the application core is responsible for starting the network core and connecting its GPIO pins.
-   Therefore, to run any sample on nRF5340, the application core must be programmed, even if the firmware is supposed to run only on the network core, and the firmware for the application core must set :option:`CONFIG_BOARD_ENABLE_CPUNET` to ``y``.
-   You can use the :ref:`nrf5340_empty_app_core` sample for this purpose.
-   For details, see the code in :file:`zephyr/boards/arm/nrf5340dk_nrf5340/nrf5340_cpunet_reset.c`.
+   The above list might not be exhaustive.
+
+Samples that do not need radio communication can run on the application core of the nRF5340.
+They do not require any firmware on the network core.
+Therefore, the network core can remain empty.
+
+If you want to enable the network core anyway, you can do this by setting :option:`CONFIG_BOARD_ENABLE_CPUNET` in the image for the application core.
+
+.. _ug_nrf5340_multi_image:
 
-To program only the application core, follow the instructions in :ref:`gs_programming_ses` and use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuappns`` as build target.
+Single image vs. multi-image build
+**********************************
 
-When programming both the application core and the network core, you can choose whether you want to build and program both images separately or combined as a :ref:`multi-image build <ug_multi_image>`.
+If a sample consists of two images (one for the application core and one for the network core), these images can be built separately or combined as a :ref:`multi-image build <ug_multi_image>`, depending on the sample configuration.
 
 In a multi-image build, the image for the application core is the parent image, and the image for the network core is treated as a child image in a separate domain.
```

Comment:
This is incorrect, the network core sample can also be the parent image, as shown in the `empty_app_core` comment above.